src/main/resources/plugin.json

```diff
@@ -2,7 +2,7 @@
     "name": "Chunky AOV",
     "author": "Redox",
     "main": "dev.thatredox.chunkyaov.ChunkyAOV",
-    "version": "0.0.1",
-    "targetVersion": "2.4.2",
+    "version": "0.0.2",
+    "targetVersion": "2.4.4",
```

Comment:
We shouldn't increase the target version (technically we are still targeting 2.4.0)